GLTFSDK.Test/Source/TestUtils.h

```diff
@@ -135,6 +138,25 @@ namespace Microsoft
                 auto json = std::string(std::istreambuf_iterator<char>(*input), std::istreambuf_iterator<char>());
                 return json;
             }
+
+            inline Document CheckExtensionRoundTripEquality(const char* relativePath)
```

Comment:
I think this makes more sense as a function in `GLTFExtensionsTests.cpp` in the anonymous namespace. This isn't actually a utility function like the rest of these functions.